mechanics: Rewording and additions (#4)
gonX/dustforce-wiki: website/_mechanics/basics/02-terminology.md

{:.no_toc}
### Superdrop
Unrelated to actual Supers, this is the technique used for quickly cleaning dust (using light attacks) while falling at high speeds.

{:.no_toc}
### Left friction

> An unintentional programming oversight causes air friction to be much more intense when travelling leftwards.

Author: ```suggestion
An unintentional programming oversight causes air friction to be much more intense when travelling left.
```
Suggestion by C

A tech that enters the wrong state when jumping, allowing an earlier wallrun.

See [Raiserunning](#raiserunning)

{:.no_toc}
### Clip
Certain configurations of geometry allow the player to pass through solid walls.

{:.no_toc}
### Magnet

> Jumping mid-air will "magnet" you to walls approximately within half of the width (TODO: fact check?) of your characters hitbox.

Reviewer: Fact Check: Wall magnets work up to 41 - 42 units away, almost a full tile. differences due to symmetry bugs.

Author: ```suggestion
Jumping mid-air will "magnet" you to walls up to 42 horizontal units away (almost a full tile).
```
Something like this?

This is often negatively experienced for the first time for many players in [Backup Shift](#maps-level-backup-shift) (TODO: earlier levels?)

> 

Reviewer: Possibly mention how Wall Magnets can be used to positive effect. for example, Snap Jumps / Snap Dashes (alternate names for intentional wall-magneting) can make Giga Walls easier

Author: ```suggestion
However, it can also be used for positive effect as is often seen in the Walls (last) section of [Giga Difficult](#maps-level-giga-difficult).
```
